tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -160,6 +155,7 @@ spec:
 
     - name: validate
       image: quay.io/enterprise-contract/ec-cli:snapshot
+      onError: continue
```

Comment:
Will the Task still fail if there's an error, e.g. invalid policy config is provided? I think so.

Reply:
What's the reason for adding this line? Maybe a comment is worthwhile.

Reply:
Added a comment, `ec` could fail, for example because bad input was provided and the rest of the steps would be skipped, including the output of debug logs so we would loose valuable clues as to how to debug the issue